appveyor.yml

```diff
@@ -86,6 +98,27 @@ test_script:
 
       if ($testsExitCode -ne 0) {exit $testsExitCode}
 
+  - set CLCACHE_MEMCACHED=127.0.0.1:11211
+  - ps: |
+      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache --cov=storage
+      $testsExitCode = $lastexitcode
+      & coverage report
+      & coverage xml
+
+      $wc = New-Object 'System.Net.WebClient'
+      $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\integrationtests.xml))
+
+      & codecov --no-color -X gcov -F integrationtests -e PYTHON_INSTALL
```

Comment:
I think it would be nice to use a separate flag here, i.e. instead of `-F integrationtests` maybe use `-F integrationtests_memcached`. The benefit of this is that one can discriminate the memcached-specific portions more clearly in the coverage report. I think(!) that without this, you'll overwrite the coverage report for the earlier upload with the `integrationtests` flag (i.e. the run without having memcached enabled).